docs/source/format/Columnar.rst

```diff
@@ -1019,7 +1024,13 @@ Schematically we have: ::
 In the file format, there is no requirement that dictionary keys
 should be defined in a ``DictionaryBatch`` before they are used in a
 ``RecordBatch``, as long as the keys are defined somewhere in the
-file.
+file. Further more, it is invalid to have more then one non-delta
+dictionary batch per dictionary ID.  Delta dictionaries are applied
+in the order they appear in the file footer.
```

Comment:
If we're allowing delta dictionaries in the file format at all, then why can there only be one? 

If you're reading the file, it seems you could assemble all the deltas during the initial pass to create the "master" dictionary

Reply:
> If we're allowing delta dictionaries in the file format at all, then why can there only be one?
I agree.  I added emphasis on what I think I wrote.  This only allows for one "non-delta" dictionary.  
> Further more, it is invalid to have more then one **non-delta** dictionary batch per dictionary ID

Do you think we Should we disallow Delta dictionaries?